src/Component/Config/Builder/Field.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Grid\Config\Builder;
+
+final class Field implements FieldInterface
+{
+    private string $name;
+    private string $type;
+    private ?string $path = null;
+    private ?string $label = null;
+    private ?bool $enabled = null;
+    /** @var bool|string|null */
+    private $sortable = null;
+    private ?int $position = null;
+    private array $options = [];
+
+    public function __construct(string $name, string $type)
+    {
+        $this->name = $name;
+        $this->type = $type;
```

Comment:
I'm just wondering if type should have predefined set of values

Reply:
Predefined set of values are on this next PR:
https://github.com/Sylius/SyliusGridBundle/pull/173